gen.sh

```diff
@@ -7,16 +7,15 @@ has() {
         esac
 }
 cd "${gitd}"
-if [ ! -f ./discordemojimap/mapping.go ]; then if has git; then git submodule update --init; else echo "FAIL, git missing, please find your own way to download the submods"; exit 1;fi;fi
+if [ ! -d ./blobmoji/png ]; then if has git; then git submodule update --init; else echo "FAIL, git missing, please find your own way to download the submods"; exit 1;fi;fi
 # git isn't ""REALLY"" needed, you just have to trick the script into finding the submods
-cp ./discordemojimap/mapping.go ./
 if ! has sed; then echo "sed NOT FOUND!!"; exit 1; fi
 sed -e '/\/\//d' -e '/u/,$!d' -e 's/"//g' -e 's/:[[:space:]]/=/g' -e 's/\\//g' -e 's/,//g' -e 's/\t//g' -e 's/}//g' -e '/^$/d' -i ./mapping.go
 while IFS='=' read -r name val; do # Thx Perish :)
 	IFS=''
 	printf 'img[alt=":%s:"],li[aria-label^=":%s:"]>div{content:var(--a);background-image:var(--a)!important;--a:url("data:image/png;base64,%s");background-size:contain;object-position: var(--op);--op:-9999px -9999px;}\n' "${name}" "${name}" "$(base64 blobmoji/png/128/emoji_${val}.png | tr -d '\n')"
 done < ./mapping.go
-grep "//" ./discordemojimap/mapping.go | sed -e 's/:.*//g' -e 's/[[:blank:]]//g' -e 's/"//g' >> temp1
+grep "//" mapping.go | sed -e 's/:.*//g' -e 's/[[:blank:]]//g' -e 's/"//g' >> temp1
```

Comment:
The issue with this is that the sed call on line 13 removes all lines with comments, so this would do nothing...
But the missing file is still needed incase I decide to fix #1 

I'll have to think about a proper way of keeping it...